lib/Migration/Version0001Date20181211222142.php

```diff
@@ -62,27 +66,28 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		$table = $schema->getTable(CoreRequestBuilder::TABLE_CACHE_ACTORS);
 		$table->changeColumn(
 			'source', [
-						'notnull' => true,
-						'length'  => CoreRequestBuilder::SOURCE_LENGTH,
+						'notnull' => false,
+						'type'    => Type::getType(Type::TEXT)
```

Comment:
if I remember correctly changing type is not supported on all DBMS.
So you have to create a new column and copy over the values and then delete the old one.

See https://github.com/nextcloud/server/pull/12275/files#diff-1307110df255230fc590b38bbc452bc1 for the simplest example

Reply:
improved